cirq-core/cirq/devices/named_topologies.py

```diff
@@ -281,14 +290,25 @@ def get_placements(
 def draw_placements(
     big_graph: nx.Graph,
     small_graph: nx.Graph,
-    small_to_big_mappings,
-    max_plots=20,
+    small_to_big_mappings: Sequence[Dict],
+    max_plots: int = 20,
     axes: Sequence[plt.Axes] = None,
+    tilted=True,
 ):
     """Draw a visualization of placements from small_graph onto big_graph using Matplotlib.
 
     The entire `big_graph` will be drawn with default blue colored nodes. `small_graph` nodes
     and edges will be highlighted with a red color.
+
+    Args:
+        big_graph: A larger graph to draw with blue colored nodes.
```

Comment:
Nit: parent graph ?

Reply:
that would be a breaking change and less clear of a name, imo